src/shared/pages/team/teamSearch/teamSearch.tsx

```diff
@@ -43,14 +56,27 @@ const TeamSearch: React.FC<TeamSearchProps> = ({ history }) => {
             <h1 className="govuk-heading-xl">
                 Team search
             </h1>
-            <TypeAhead
-                clearable={true}
-                disabled={false}
-                getOptions={getTeamsForTypeahead}
-                label={'Teams'}
-                name={'Teams'}
-                onSelectedItemChange={setSelectedTeam}
-            />
+            <div className="govuk-grid-row">
+                {!showInactive && <div className="govuk-grid-column-three-quarters"><TypeAhead
```

Comment:
Can these be pulled into one? 

`getOptions={!showInactive ? getTeamsForTypeahead(getTeams : getTeamsForTypeahead(getAllTeams)}`

Other alternatives might be available...